### ISSUE TITLE
Hello i have now facing with this problem that i have install nodemon everything file but always need to save the file where i have made changes to appear the changes in website

### ISSUE THREAD

**Issue by sansanaye99** [remy/nodemon#2221]
- Versions: 
- `nodemon -v`:
- Operating system/terminal environment (powershell, gitshell, etc): 
- Using Docker? What image: 
- Command you ran:

### Steps to reproduce

---

If applicable, please append the `--dump` flag on your command and include the output here **ensuring to remove any sensitive/personal details or tokens**.

**Comment by remy** [comment]
Can you rephrase and explain what issue you're seeing?
